src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -141,10 +152,27 @@ public void parse_allFieldsSpecified_success() {
         assertParseSuccess(parser, userInput, expectedCommand);
     }
 
+    @Test
```

Comment:
Again, think a test case for failing by name would be good! Person not in model, invalid name, etc.

Reply:
added!